### PR TITLE
Navigation icon and style refresh

### LINKED ISSUE #13981: Refine Toolbar UI with New Icons and styling refinements
Issue: Dreams of UI refinements stuck in design world. 

Fix: Implement collection of UI design refinements and new icon set.

Test proceedure:

Open new instance that includes new changes.
Compare to previous version.
Be amazed by beauty of new version.
Click buttons and resize window to make sure nothing blows up or looks wonky.

actual screen shot showing changes from @petemill go here:

### PR DESCRIPTION
**Pull request by petemill**
Fix #13981 

Adjusts much of the top nav / url bar area including:
- Buttons get active state and updated hover state
- Buttons get new icons
- Url bar gets alignment and height change
- Url bar titlemode only swaps the url for the page title - it does not hide the buttons
- Url bar gets a loading progress bar background
- Payments / Funding button gets same states as other navigation buttons: hover, active, disabled, checked (cc @NejcZdovc @ryanml )
- Tab gets shape change
- Tab loading icon is more solid
- Tab page next / prev page navigation button gets new icons
- Tab preview is less "poppy" (and content area is less faded)
- Preferences navigation icons updated

See #13981 for visual details and evolution.

This PR represents the third round of feedback and I feel is sufficient to be merged for the next beta release in order to test other release features with these style changes merged. Subsequent feedback based on that can be covered by a subsequent PR.

## Submitter Checklist:

- [x] Submitted a [ticket](https://github.com/brave/browser-laptop/issues) for my issue if one did not already exist.
- [x] Used Github [auto-closing keywords](https://help.github.com/articles/closing-issues-via-commit-messages/) in the commit message.
- [x] Added/updated tests for this change (for new code or code which already has tests).
- [ ] Ran `git rebase -i` to squash commits (if needed).
- [x] Tagged reviewers and labelled the pull request [as needed](https://github.com/brave/browser-laptop/wiki/Pull-request-process).
- [x] Request a security/privacy review [as needed](https://github.com/brave/handbook/blob/master/development/security.md#how-to-request-a-security-review). (Ask a Brave employee to help if you cannot access this document.)

## Test Plan:
 - Confirm buttons and URL bar work as expected
 - Confirm styles look as expected

## Reviewer Checklist:

- [ ] Request a security/privacy review [as needed](https://github.com/brave/handbook/blob/master/development/security.md#how-to-request-a-security-review) if one was not already requested.

Tests

- [ ] Adequate test coverage exists to prevent regressions
- [ ] Tests should be independent and work correctly when run individually or as a suite [ref](https://github.com/brave/browser-laptop/wiki/Code-Guidelines#test-dependencies)
- [ ] New files have MPL2 license header

### CODE REVIEW

**Comment by petemill** [comment]
@ryanml are those issues reproducible on current master? Or new with this branch?

**Comment by ryanml** [comment]
@petemill cannot repro on master, no

**Comment by petemill** [comment]
After discussion with @ryanml I've re-pushed, bringing back the old functionality of not showing a site as included until it has met it's minimum visit duration and count, as well as updating the tests to support that.

Ready for review @bsclifton.

**Comment by bsclifton** [comment]
Updates to the URL bar look fantastic- however, the green is SUPER bright IMO. It's a little hard to read (ex: I don't think there's enough contrast)
screen shot 2018-06-14 at 1 38 58 pm

**Comment by petemill** [comment]
We’ve gone through 3 rounds of feedback and changes for this and that
hasn’t been requested to be changed on any of those rounds. Color was
chosen from a branding perspective and validated on security review also
against accessibility requirements. Can we get a decision from @bradrichter?

File: app/renderer/components/navigation/urlBarIcon.js [hunk 1]
```diff
@@ -174,13 +108,85 @@ class UrlBarIcon extends React.Component {
       props.onDragStart = this.onDragStart
     }
 
-    return <span
+    let icon = null
+    let iconTestId = ''
+    let isInsecure = false
+    let isExtendedSecure = false
+    if (this.props.activateSearchEngine) {
+
```
Comment (bsclifton) [inline app/renderer/components/navigation/urlBarIcon.js]: intentionally blank?

**Comment by srirambv** [comment]
Icons looks relatively smaller than the current ones. Is this intentional?

File: app/renderer/components/navigation/urlBarIcon.js (hunk 1 above)
Comment (petemill) [reply app/renderer/components/navigation/urlBarIcon.js]: Addressed, thanks for the spot

**Comment by petemill** [comment]
Made the following changes:
[x] Addressed syntactic feedback (empty if condition), thanks @bsclifton 
[x] Shields icon in Preferences Navigation is now the Brave logo again, this time full color - thanks @rossmoody

**Comment by petemill** [comment]
0.23.x 59339887d

**Pull request merged**